### PR TITLE
Bump to 3.42.0

### PR DESCRIPTION
**Pull request by SteinerOk**

### CODE REVIEW

File: sqlite-android/build.gradle [hunk 1]
```diff
@@ -48,7 +48,7 @@ android {
 
 dependencies {
     api 'androidx.core:core:1.9.0'
-    api 'androidx.sqlite:sqlite:2.3.0'
+    api 'androidx.sqlite:sqlite:2.3.1'
```
Comment (LeoColman) [inline sqlite-android/build.gradle:51]: This is a version bump that seems to be going in accidentally, is that intended?

File: sqlite-android/build.gradle [hunk 2]
```diff
@@ -57,7 +57,7 @@ dependencies {
 }
 
 ext {
-    sqliteDistributionUrl = 'https://www.sqlite.org/2023/sqlite-amalgamation-3410100.zip'
+    sqliteDistributionUrl = 'https://www.sqlite.org/2023/sqlite-amalgamation-3410200.zip'
```
Comment (LeoColman) [inline sqlite-android/build.gradle:60]: This is a version bump that seems to be going in accidentally, is that intended?

File: sqlite-android/build.gradle (hunk 1 above)
Comment (SteinerOk) [reply sqlite-android/build.gradle:51]: @LeoColman that intended, update of AndroidX SQLite - [https://developer.android.com/jetpack/androidx/releases/sqlite#2.3.1](url)

**Comment by SteinerOk** [comment]
Hey, @npurushe may you review this PR, please :)

**Comment by npurushe** [comment]
thanks for submitting :)

**Comment by SteinerOk** [comment]
@npurushe Oh, I think you set incorrect tag for release, I update Pr to version 3.42.0, but you set 3.41.2

**Pull request merged**